.ai-agents/memory-bank/activeContext.md

```diff
@@ -158,6 +154,8 @@ The current focus is on enhancing the Reviewer User experience with AI-driven an
    - Properly type nested relationship data
    - Use optimized queries with nested joins
    - Cache query results when they'll be reused
+   - Follow migration guidelines in `docs/migrationOpsContext.md` when creating database migrations
+   - Adhere to schema design patterns in `docs/schemaPatterns.md` for consistent database design
```

Comment:
Knowledge is referenced. The purpose is to verify if it can be used not only for review but also for build.